src/main/java/me/totalfreedom/totalfreedommod/rank/RankManager.java

```diff
@@ -152,13 +158,13 @@ public void onPlayerJoin(PlayerJoinEvent event)
             player.getInventory().clear();
             player.setOp(false);
             player.setGameMode(GameMode.SURVIVAL);
-            plugin.pl.getPlayer(player).getFreezeData().setFrozen(true);
+            plugin.pl.getPlayer(player).setEditBlocked(true);
             player.sendMessage(ChatColor.RED + "You are marked as an impostor, please verify yourself!");
             return;
         }
 
         // Set display
-        if (isAdmin || FUtil.DEVELOPERS.contains(player.getName()))
+        if (isAdmin || FUtil.DEVELOPERS.contains(player.getName()) || FUtil.FOUNDER.contains(player.getName()))
```

Comment:
I mean currently, if you are an owner, how is the login message handled? Because its not in this list here...